models/stg_jira__issue.sql

```diff
@@ -40,7 +44,11 @@ final as (
         project as project_id,
         reporter as reporter_user_id,
         resolution as resolution_id,
-        resolved as resolved_at,
+        {% if target.type == 'redshift' -%}
+            cast(resolved as timestamp without time zone)
+        {% else -%}
+            resolved
+        {% endif %} as resolved_at,
         status as status_id,
         status_category_changed as status_changed_at,
```

Comment:
I think the reason it didn't fail before with the casting not being included is because the `status_changed_at` field is not used in any downstream functions that this could error out in. Since `timestamp` and `timestamptz` are _technically_ the same, they are just interpreted differently I felt it would be fine to not explicitly cast these fields.

I did go down the thought experiment of casting these, but then we would be explicitly casting every time type field in the package. But because the datatypes are the same in nature, but differ in interpretation. It would be okay to only cast the ones used in downstream date functions. 

Thoughts?

Reply:
That makes sense to me. We can revisit later if it becomes an issue (which sounds like it won't). Ow, the rest lgtm!